Wrappers/Python/cil/optimisation/functions/TotalVariation.py

```diff
@@ -54,6 +54,8 @@ class TotalVariation(Function):
       :type split: bool, default `False`           
       :param info: force a print to screen stating the stop
       :type info: bool, default `False`
+      :param strongly_convex_constant: Adds a strongly convex function to the TotalVariation functional.
```

Comment:
I suggest the name `strong_convexity_constant`, which sounds more natural to me: with "strongly convex constant" it is the constant that is strongly conxex, and not a constant of strong convexity :)

Reply:
I changed to `strong_convexity_constant`.